### PR TITLE
#82 | Event Api

### PR DESCRIPTION
**Pull request by melzareix**

### CODE REVIEW

File: app/Http/Controllers/EventsAPIController.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Photo;
+use App\Event;
+
+class EventsAPIController extends Controller
```
Comment (AhmadElsagheer) [inline app/Http/Controllers/EventsAPIController.php]: **EventAPIController**

File: app/Http/routes.php [hunk 2]
```diff
@@ -251,3 +251,13 @@
     //show an organization, its events, reviews, subscribers
     Route::get('api/organization/{id}' , 'OrganizationAPIController@show');
 
+/*
+|--------------------------
+| Events API Routes
+|--------------------------
+*/
+//get a list of all organizations
```
Comment (AhmadElsagheer) [inline app/Http/routes.php]: organizaton?

**Comment by omarradwan** [comment]
Hahahahahahaha 😂😂

**Comment by melzareix** [comment]
Fixed.

**Pull request merged**